.changeset/big-beers-reflect.md

```diff
@@ -0,0 +1,23 @@
+---
+'@commercetools-uikit/localized-multiline-text-field': minor
```

Comment:
I think before merging (noticed after submitting review) we should make this a patch? It's not really a new feature we're rolling out unless we calling it: finally we add correct markup for error messages.

Reply:
[done](https://github.com/commercetools/ui-kit/pull/2950/commits/e70c1c470a263e16f9bbeb1c7c76ed1330d4cb4a)